lib/thrift/protocols/binary.ex

```diff
@@ -36,6 +36,18 @@ defmodule Thrift.Protocols.Binary do
   def int_type({:set, _}), do: 14
   def int_type({:list, _}), do: 15
 
+  def exception_type(0), do: :unknown
```

Comment:
This should match `_` (and go at the bottom of the list). Spec says: "used in case the type from the peer is unknown".